Task 3/Task3.2.1/Task3.2.1/Program.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Task_2._1;
+
+namespace Task3._2._1_
+{
+    class Program
+    {
+       
+        
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите текст");
+            string[] wordArr, wordUnique;
+            Input(out wordArr, out wordUnique);
+
+            WordInText[] wordInTexts = new WordInText[wordUnique.Length];
+            for (int i = 0; i < wordUnique.Length; i++)
+            {
+                wordInTexts[i].Word = wordUnique[i];
+                wordInTexts[i].Count = CountWord(wordArr, wordInTexts[i].Word);
+            }
+            WordInTextSort(wordInTexts);
```

Comment:
You could use GroupBy linq method or use a Dictionary to go through words collection only one time